src/schema/auction_result.js

```diff
@@ -3,9 +3,12 @@ import { amount } from "./fields/money"
 import { IDFields, NodeInterface } from "./object_identification"
 import { GraphQLFloat, GraphQLNonNull, GraphQLString, GraphQLObjectType, GraphQLEnumType } from "graphql"
 import { connectionDefinitions } from "graphql-relay"
-import { has } from "lodash"
+import { has, indexOf } from "lodash"
 import Image from "schema/image"
 
+const currencyCodes = require("./../lib/currency_codes.json")
```

Comment:
I wish we could leave a comment in the JSON file, but lacking that, maybe a link to the source for this data here?

Reply:
we could switch to using `file.readFile` && `JSON5.parse` instead to support comments in JSON

Reply:
interesting! I'm not sure if its worth adding another dep (`json5`- that would be a separate lib? https://github.com/json5/json5 seems to indicate yes) for that, ill add a comment here though, for sure.